test/WireMock.Net.Tests/WireMockServer.Admin.cs

```diff
@@ -404,5 +405,53 @@ public void WireMockServer_Admin_AddMappingsAndSaveToFile()
             staticMappingHandlerMock.Verify(m => m.FolderExists("folder"), Times.Once);
             staticMappingHandlerMock.Verify(m => m.WriteMappingFile(Path.Combine("folder", guid + ".json"), It.IsAny<string>()), Times.Once);
         }
+
+        [Fact]
+        public async void WireMockServer_Admin_DeleteMappings()
+        {
+            // Arrange
+            var server = WireMockServer.Start(new WireMockServerSettings
+            {
+                AllowBodyForAllHttpMethods = true,
+                StartAdminInterface = true,
+                ReadStaticMappings = false,
+                WatchStaticMappings = false,
+                WatchStaticMappingsInSubdirectories = false
+            });
+            string folder = Path.Combine(GetCurrentFolder(), "__admin", "mappings", "delete_mappings");
+            server.ReadStaticMappings(folder);
```

Comment:
Instead of reading static mapping in this test, I think it's better to just add 3 new mappings to the server and then delete 2 mappings using tghe new delete method..